Reference/google/protobuf/any_test.pb.swift

```diff
@@ -66,20 +66,18 @@ struct ProtobufUnittest_TestAny: SwiftProtobuf.Message, SwiftProtobuf._MessageIm
 
     init() {}
 
-    func copy() -> _StorageClass {
-      let clone = _StorageClass()
-      clone._int32Value = _int32Value
-      clone._anyValue = _anyValue
-      clone._repeatedAnyValue = _repeatedAnyValue
-      return clone
+    init(storage source: _StorageClass) {
```

Comment:
Yeah, I don't have a strong preference for either one, honestly.

@thomasvl originally had `copying:` anyway, so let's go with that :)